test/Twilio.Test/Clients/TwilioRestClientTest.cs

```diff
@@ -113,5 +113,15 @@ public void TestRedirectResponse()
             TwilioRestClient twilioClient = new TwilioRestClient("foo", "bar", null, null, client);
             twilioClient.Request(request);
         }
+
+        [Test]
+        public void TestActivatingDebugLogging()
+        {
+            client.MakeRequest(Arg.Any<Request>()).Returns(new Response(HttpStatusCode.OK, "OK"));
+            Request request = new Request(HttpMethod.Get, "https://www.contoso.com");
+            TwilioRestClient twilioClient = new TwilioRestClient("foo", "bar", null, null, client);
+            twilioClient.LogLevel = "debug";
```

Comment:
Why are there no asserts here?

Reply:
This test sets LogLevel on the _TwilioRestClient_ as debug but the only outcome is the logging output. What should I be asserting on?

Reply:
You should make a request and assert that the logging output exists and/or contains the "expected" value/content